config/routes.rb

```diff
@@ -55,7 +55,11 @@
             get 'content'
           end
         end
-        resources :arf_reports, :only => [:index, :show, :destroy]
+        resources :arf_reports, :only => [:index, :show, :destroy] do
```

Comment:
I believe we have it [disabled](https://github.com/theforeman/foreman_openscap/blob/master/.rubocop.yml#L15-L17), is it possible hound is ignoring it?